src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -2749,6 +2749,10 @@ public override void Visit(FirstNameNode node)
                     {
                         _txb.NodesToReplace.Add(new KeyValuePair<Token, string>(node.Token, entity.EntityName));
                     }
+                    else if(lookupInfo.Data is NameSymbol nameSymbol)
```

Comment:
>NameSymbol [](http://example.com/codeflow?start=47&length=10)

Should NameSymbol just implement IExternalEntity? 
(and then this check wouldn't be necessary)